identity-appsupport/src/commonMain/kotlin/com/android/identity/appsupport/ui/x509chain/X509ChainViewer.kt

```diff
@@ -0,0 +1,108 @@
+package com.android.identity.appsupport.ui.x509chain
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.foundation.pager.HorizontalPager
+import androidx.compose.foundation.pager.rememberPagerState
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import com.android.identity.crypto.X509CertChain
+
+private const val TAG = "X509ChainViewer"
+private val PAGER_INDICATOR_HEIGHT = 30.dp
+private val PAGER_INDICATOR_PADDING = 8.dp
+
+@Composable
+/**
+ * Display Pager for the list of certs, with corresponding List of additional warnings and infos.
+ * composables.
+ *
+ * @param infosAndWarnings - assuming the Chain list can be matched by the same sequence chain
+ *                           of needed Composables 1:1 displayed at the top of the details page.
+ * */
```

Comment:
`* */` looks a bit weird, any reason for that? The rest of the codebase uses just `*/`, suggest to use that...